src/colorizer/LightColorizer.cpp

```diff
@@ -261,9 +262,110 @@ void LightColorizer::SetSOs(std::vector<Sombrero::FastColor> const& colors) {
 void LightColorizer::Refresh() {
     auto const& colors = getColor();
     SetSOs(colors);
+    
```

Comment:
Change `UnityEngine::Color` to `Sombrero::FastColor` and use `Alpha()` instead of `ColorExtensions